{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":20,"msg":"Sync server version: 1.2.4 2020-09-14 17:52:19 -0500 0aa4a9d288d4ecef4183f4fafe59fb95da7ae967","time":"2021-04-24T16:55:28.525Z","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"msg":"Acquiring actors for session 1619283329: ACTOR1","time":"2021-04-24T16:55:28.537Z","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"msg":"Session 1619283329 with label \"web 5\" started at 2021-04-24 16:55:28 UTC","time":"2021-04-24T16:55:28.549Z","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.23Z","msg":"Actor 59927 of type ACTOR1 was acquired by test session 1619283329","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":20,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.285Z","msg":"DEBUG: Test actor JAR versions:","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":20,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.291Z","msg":"DEBUG:   opentest-appium-0.0.1.jar: 2020-09-14 17:52 -0500 0.0.1-0aa4a9d288d4ecef4183f4fafe59fb95da7ae967","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":20,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.296Z","msg":"DEBUG:   opentest-base-0.0.1.jar: 2020-09-14 17:52 -0500 0.0.1-0aa4a9d288d4ecef4183f4fafe59fb95da7ae967","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":20,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.299Z","msg":"DEBUG:   opentest-selenium-0.0.1.jar: 2020-09-14 17:52 -0500 0.0.1-0aa4a9d288d4ecef4183f4fafe59fb95da7ae967","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":40,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.314Z","msg":"WARN: Failed to log extension JARs","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.528Z","msg":"==================== ACTOR1: web/Find the React repo on GitHub's website ====================","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.528Z","msg":"Actor ACTOR1 started executing test web/Find the React repo on GitHub's website...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.623Z","msg":"Executing segment 1 of test web/Find the React repo on GitHub's website...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.625Z","msg":"Evaluating action 1/4 (org.getopentest.selenium.NavigateTo) in segment 1 (Navigate to the google homepage)...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":10,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.627Z","msg":"TRACE: Initializing action org.getopentest.selenium.NavigateTo...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.629Z","msg":"Using local Selenium server","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":50,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:33.695Z","msg":"ERROR: java.lang.Exception: Failed executing test web/Find the React repo on GitHub's website, segment 1\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2165)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:2981)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3017)\r\n\tat org.getopentest.Main.main(Main.java:20)\r\nCaused by: java.lang.RuntimeException: There was an error while executing action no. 1 (org.getopentest.selenium.NavigateTo) in segment 1 of test web/Find the React repo on GitHub's website.\nThe action definition was:\n{\n  \"action\": \"org.getopentest.selenium.NavigateTo\",\n  \"args\": {\n    \"url\": \"https://www.google.com\"\n  },\n  \"description\": \"Navigate to the google homepage\"\n}\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2123)\r\n\t... 3 more\r\nCaused by: java.lang.RuntimeException: Failed executing action org.getopentest.selenium.NavigateTo with arguments {\"url\":\"https://www.google.com\"}\r\n\tat org.getopentest.base.TestActor.executeActionByDef(TestActor.java:1821)\r\n\tat org.getopentest.base.TestActor.executeTestSegment(TestActor.java:2073)\r\n\t... 3 more\r\nCaused by: java.lang.RuntimeException: java.lang.RuntimeException: Failed to find the Selenium driver executable for browser \"chrome\". File \"C:/Selenium/chromedriver.exe\" does not exist. You can provide the correct path to the driver in the test actor's configuration file.\r\n\tat org.getopentest.base.TestActor.executeActionByClassName(TestActor.java:1626)\r\n\tat org.getopentest.base.TestActor.executeActionByDef(TestActor.java:1705)\r\n\t... 4 more\r\nCaused by: java.lang.RuntimeException: Failed to find the Selenium driver executable for browser \"chrome\". File \"C:/Selenium/chromedriver.exe\" does not exist. You can provide the correct path to the driver in the test actor's configuration file.\r\n\tat org.getopentest.selenium.core.SeleniumHelper.setDriverExecutable(SeleniumHelper.java:306)\r\n\tat org.getopentest.selenium.core.SeleniumHelper.createDriver(SeleniumHelper.java:67)\r\n\tat org.getopentest.selenium.core.SeleniumHelper.getDriver(SeleniumHelper.java:237)\r\n\tat org.getopentest.selenium.core.SeleniumTestAction.initialize(SeleniumTestAction.java:203)\r\n\tat org.getopentest.base.TestActor.executeAction(TestActor.java:1509)\r\n\tat org.getopentest.base.TestActor.executeActionByClassName(TestActor.java:1603)\r\n\t... 5 more\r\n","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:34.231Z","msg":"Test session has completed","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":10,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:34.233Z","msg":"TRACE: Firing TEST_COMPLETED event...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:34.236Z","msg":"Using local Selenium server","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":50,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:34.238Z","msg":"ERROR: The \"runOneSession\" method failed","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":50,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:34.239Z","msg":"ERROR: java.lang.NullPointerException\r\n\tat org.getopentest.selenium.core.SeleniumTestAction$1.update(SeleniumTestAction.java:190)\r\n\tat java.util.Observable.notifyObservers(Unknown Source)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3006)\r\n\tat org.getopentest.base.TestActor.runOneSession(TestActor.java:3017)\r\n\tat org.getopentest.Main.main(Main.java:20)\r\n","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":30,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:55:34.242Z","msg":"Actor 59927 of type ACTOR1 with no tags is waiting to be acquired by a test session...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":10,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:56:58.038Z","msg":"TRACE: Initializing test session information in the \"announce\" method...","v":0}
{"name":"1619283329","hostname":"QuadTMO-008","pid":45804,"level":10,"actorType":"ACTOR1","actorId":"59927","time":"2021-04-24T16:56:58.038Z","msg":"TRACE: Session ID in sync server response was 1619283413; actor session ID was null","v":0}
